src/Cache/ManifestCache.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+/**
+ * File containing an abstract class for holding Manifest Cache functionality.
```

Comment:
This file is a concrete implementation of the cache, so this docblock is not accurate.